### PR TITLE
Update password-policy-recommendations.md

### PR DESCRIPTION
**Pull request by Crys-7**
Maintain an fourteen -character minimum length requirement

### CODE REVIEW

**Comment by Crys-7** [comment]
Updating:   Maintain an fourteen -character minimum length requirement

File: microsoft-365/admin/misc/password-policy-recommendations.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ Good password practices fall into a few broad categories:
 The primary goal of a more secure password system is password diversity. You want your password policy to contain lots of different and hard to guess passwords. Here are a few recommendations for keeping your organization as secure as possible.
   
 
-- Maintain an eight-character minimum length requirement
+- Maintain an fourteen -character minimum length requirement
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/misc/password-policy-recommendations.md]:
```suggestion
- Maintain a fourteen-character minimum length requirement
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Crys-7, kindly commit the suggested changes. Many thanks.

**Comment by Crys-7** [comment]
committed

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@kwekuako Gentle reminder.

**Pull request merged**